github/event_types.go

```diff
@@ -175,6 +175,7 @@ type IntegrationInstallationEvent struct {
 	Action       *string       `json:"action,omitempty"`
 	Sender       *User         `json:"sender,omitempty"`
 	Installation *Installation `json:"installation,omitempty"`
+	HTMLURL      *string       `json:"html_url,omitempty"`
```

Comment:
As discussed in #591, please move this line to the `type Installation struct` in `github/integration_installation.go`.
Thank you, @liubinyi!